src/v2/controllers/FormController.js

```diff
@@ -147,7 +149,19 @@ export default Controller.extend({
     }
 
     this.toggleFormButtonState(true);
+
+    // NOTE: okta-idx-js does not know whether remediations
+    // are required to redirect or perform a GET request.
+    // As a result, we handle this accordingly by checking the formName
+    const fullPageRedirectForms = [ FORMS.REDIRECT_IDP ];
```

Comment:
we are opening a pandora box here, where any form can be added into this array that can redirect.. We need some kind of restriction for developers not to make that mistake.